### ISSUE TITLE
"#Covid-19" content is a bug and should be a `h3` without the "#"

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#195]
## Description

On several event pages there is a covid-19 warning that starts with a "#". This is a typo and should be a `h3`.

<img width="688" alt="Screen Shot 2022-09-29 at 8 39 33 PM" src="https://user-images.githubusercontent.com/14154792/193165702-dbd5f7fb-5c66-4579-a7c5-8ffa0a57f047.png">

## Success criteria

- [x] Remove the "#" 
- [x] Change the `strong` element to a `h3`

<img width="679" alt="Screen Shot 2022-09-29 at 8 41 12 PM" src="https://user-images.githubusercontent.com/14154792/193165859-cdf3f890-7ee0-45e3-a7e4-367771ceadc5.png">
